### PR TITLE
Fix 1079 Message is displayed when user is successfully signed in

### LINKED ISSUE #1079: message should be displayed when user is successfully logged in
Summary:

There should be a message when passcode is successfully saved and user is logged in.

Steps to reproduce:

Log in using a passcode

Expected behaviour:

When passcode is successfully entered there should be a message.

Observed behaviour:

There is no message displayed when a passcode is successfully entered

### PR DESCRIPTION
**Pull request by theabhishekavi**
Fixes #1079 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java [hunk 1]
```diff
@@ -19,6 +20,7 @@ public int getLogo() {
 
     @Override
     public void startNextActivity() {
+        Toast.makeText(this, "Logged in Successfully", Toast.LENGTH_SHORT).show();
```
Comment (RajaVamsi11) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java]: Please put the hardcoded string in the strings.xml file and call from there.

Comment (theabhishekavi) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java]: @RajaVamsi11 done

Comment (RajaVamsi11) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java]: @theabhishekavi Nice.

**Comment by theabhishekavi** [comment]
@droidchef kindly review

**Comment by droidchef** [comment]
Could you please resolve the conflict ?

**Comment by theabhishekavi** [comment]
@droidchef resolved.

**Comment by theabhishekavi** [comment]
@droidchef kindly review

**Pull request merged**